docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -64,13 +92,13 @@ To enable your application to use the media controller functionality:
 
```

Comment:
At line no. 90:
Global: Please avoid using HTML tags. Use only MD specified code for hyperlinking.

Reply:
Thanks. The HTML tag changed to head anchor for MD.